src/sentry/conf/server.py

```diff
@@ -1050,12 +1050,27 @@ def create_partitioned_queues(name):
 SENTRY_TAGSTORE_OPTIONS = {}
 
 # Search backend
-SENTRY_SEARCH = os.environ.get("SENTRY_SEARCH", "sentry.search.snuba.SnubaSearchBackend")
-SENTRY_SEARCH_OPTIONS = {}
-# SENTRY_SEARCH_OPTIONS = {
-#     'urls': ['http://localhost:9200/'],
-#     'timeout': 5,
-# }
+SENTRY_USE_MORESNUBA = True
+SENTRY_SEARCH = os.environ.get("SENTRY_SEARCH", "sentry.utils.services.ServiceDelegator")
+SENTRY_SEARCH_OPTIONS = os.environ.get(
+    "SENTRY_SEARCH_OPTIONS",
+    {
+        "backend_base": "sentry.search.SearchBackend",
+        "backends": {
+            "default": {"path": "sentry.search.snuba.SnubaSearchBackend"},
+            "moresnuba": {
```

Comment:
Please let's rename this to something more self explanatory before this is merged. This is valid also for the PR that introduces MoreSnubaSearchBackend.
What about:
- GroupsDatasetSearchBackend
- CDCSnubaSearchBackend
- JoinedSnubaSearchBackend

Reply:
Renamed the class to GroupDatasetSearchTest and the path to sentry/search/snubagroups/backend.py